v6/configcat_client_test.go

```diff
@@ -76,6 +77,25 @@ func TestClient_Get(t *testing.T) {
 	c.Assert(result, qt.Equals, 3213.0)
 }
 
+func TestClient_Get_IsOneOf_Uses_Contains_Semantics(t *testing.T) {
+	c := qt.New(t)
+	srv, client := getTestClients(t)
+	srv.setResponse(configResponse{body: fmt.Sprintf(rolloutJsonFormat)})
```

Comment:
This might be easier to understand the test if the input data is inside the test
and phrased using the struct rather than the rather hard-to-read JSON encoding:

```suggestion
	srv.setResponseJSON(&rootNode{
		Entries: map[string]*entry{
			"feature": {
				Value:       false,
				VariationID: "a377be39",
				RolloutRules: []*rolloutRule{{
					Comparator:          opOneOf,
					ComparisonAttribute: "Identifier",
					ComparisonValue:     "example,foobar",
					Value:               true,
					VariationID:         "8bcf8608",
				}},
			},
		},
	})

```

More verbose but I think it's easier to see what's going on.